ethcore/src/lib.rs

```diff
@@ -15,6 +15,7 @@
 // along with Parity Ethereum.  If not, see <http://www.gnu.org/licenses/>.
 
 #![warn(missing_docs, unused_extern_crates)]
+#![cfg_attr(feature = "time_checked_add", feature(time_checked_add))]
```

Comment:
I did not find a use for this line (just tested stable and nightly with changing `checked_add` name and commenting this line: nightly pass, stable fail because of changed name).

Reply:
@sorpaas  grumble: https://github.com/paritytech/parity-ethereum/pull/10383#discussion_r265515435

Reply:
I can remove those if @sorpaas agree